.buildkite/scripts/steps/artifacts/docker_image.sh

```diff
@@ -88,12 +87,21 @@ fi
 echo "--- Build dependencies report"
 node scripts/licenses_csv_report "--csv=target/dependencies-$GIT_ABBREV_COMMIT.csv"
 
-echo "--- Upload artifacts"
+echo "--- Upload CDN assets"
+gcloud auth activate-service-account --key-file <(echo "$GCS_SA_CDN_QA_KEY")
+
+CDN_ASSETS_FOLDER=$(mktemp -d)
+tar -xf "kibana-$BASE_VERSION-cdn-assets.tar.gz" -C "$CDN_ASSETS_FOLDER" --strip=1
+
+gsutil -m cp -r "$CDN_ASSETS_FOLDER/*" "gs://$GCS_SA_CDN_QA_BUCKET/$GIT_ABBREV_COMMIT"
```

Comment:
How would we construct this path to point to the static assets from index.html? We already use a build number https://github.com/elastic/kibana/blob/main/packages/core/rendering/core-rendering-server-internal/src/bootstrap/bootstrap_renderer.ts#L82 

Is this commit hash included in any of the RawPackageInfo fields https://github.com/elastic/kibana/blob/main/packages/kbn-config/src/env.ts#L45-L48

Reply:
The build number doesn't contain all assets and in a branching scenario can cause conflicts.
```
% tree -L 3
.
├── 70129
│   └── bundles
│       ├── core
│       ├── kbn-monaco
│       ├── kbn-ui-shared-deps-npm
│       ├── kbn-ui-shared-deps-src
│       └── plugin
└── ui
    ├── favicons
    │   ├── favicon.distribution.ico
    │   ├── favicon.distribution.png
    │   ├── favicon.distribution.svg
    │   ├── favicon.ico
    │   ├── favicon.png
    │   └── favicon.svg
    ├── fonts
    │   ├── inter
    │   └── roboto_mono
    ├── legacy_dark_theme.css
    ├── legacy_dark_theme.min.css
    ├── legacy_light_theme.css
    └── legacy_light_theme.min.css

13 directories, 10 files
```

> Is this GIT_ABBREV_COMMIT hash included in any of the RawPackageInfo fields https://github.com/elastic/kibana/blob/main/packages/kbn-config/src/env.ts#L45-L48

It is, build.sha{0,12}.  It's also available to the controller as the id used to reference a build.